pkgs/tools/graphics/gmic-qt/default.nix

```diff
@@ -51,10 +49,13 @@ in stdenv.mkDerivation rec {
     chmod -R +w gmic gmic_qt
     ln -s ${CImg} CImg
 
+    cd gmic_qt
+  '';
+
+  preConfigure = ''
     cp ${gmic_stdlib} gmic/src/gmic_stdlib.h
 
     make -C gmic/src CImg.h gmic_stdlib.h
-    cd gmic_qt
```

Comment:
This should stay after build or else you have to do: `make -C ../gmic/src CImg.h gmic_stdlib.h`, which is also possible.